scripts/production/run_healthchecks.pl

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env perl
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2020] EMBL-European Bioinformatics Institute
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#      http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+=head1 NAME
+
+run_healthchecks.pl
+
+=head1 DESCRIPTION
+
+This script is a wrapper around ensj-healthcheck's run-configurable-testrunner.sh
+which accepts production_reg_conf-based configuration and automatically sets
+the correct host/port parameters for the division being tested.
+
+=head1 SYNOPSIS
+
+  perl run_healthchecks.pl $COMPARA_REG compara_curr -g ComparaAll
+
+  perl run_healthchecks.pl \
+    --reg_conf registry_configuration_file --reg_alias compara_curr \
+    -t org.ensembl.healthcheck.testcase.compara.MLSSTagGERPMSA --repair
+
+=head1 ARGUMENTS
+
+The script reads these arguments and passes the other ones straight to run-configurable-testrunner.sh
+
+=head2 DATABASE SETUP
+
+=over
+
+=item B<[--url mysql://user[:passwd]@host[:port]/dbname]>
+
+URL of the database to test
```

Comment:
```suggestion
URL of the database to test.
```
(just to follow your own style below)